content/_includes/page/list.njk

```diff
@@ -32,7 +32,7 @@
         {% if event | isFuture %}
           <strong>Upcoming!</strong>
         {% endif %}
-        [{{ event.start | getDate('iso') }}]
+        [{{ (event.start or event.date) | getDate('iso') }}]
```

Comment:
@mirisuzanne This was broken before, but I'm not sure this is the best fix. At least wanted to flag it for you.

Reply:
I'll hotfix on master